mempool/src/shared_mempool/tasks.rs

```diff
@@ -274,6 +288,7 @@ where
                             sequence_number,
                             timeline_state,
                             governance_role,
+                            transmission_state.clone(),
```

Comment:
maybe make this copyable?

Reply:
Yeah, wasn't sure where I was going with it yet, but it can be copyable.